### PR TITLE
x11-themes/kvantum: add bundled-themes use flag

### PR DESCRIPTION
**Pull request by xbjfk**
x11-themes/kvantum comes with 21 pre-installed themes, and a lot of people use kvantum only to use third-party themes. If the `bundled-themes` use flag is disabled, patch the CMakeLists.txt to remove `add_subdirectory(themes)`. The default Kvantum theme is always installed.

### CODE REVIEW

File: x11-themes/kvantum/metadata.xml [hunk 1]
```diff
@@ -5,4 +5,7 @@
     <email>thev00d00@gentoo.org</email>
     <name>Ian Whyman</name>
   </maintainer>
+  <use>
+    <flag name="bundled-themes">Build and intall the bundled themes. The base 'Kvantum' theme will allways be installed.</flag>
```
Comment (ajakk) [inline x11-themes/kvantum/metadata.xml:9]: intall?

File: x11-themes/kvantum/Manifest [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
 DIST kvantum-0.14.1.tar.gz 2461477 BLAKE2B 6c7fd45801eff1183ee46a157b5c31591da31c9b5fa9586b6095074a272fa705ac826733ce45a679b857276fbd358fdb56dc72ffe6c4744557e25c9c6782ca4a SHA512 18faa895bb6affa1744d15aa5b64636c49f7780eddca62ecee18c133702a156bca055cda473af9ea72e1d6de17fd86c4d7212bd16c544030d96d8afa4f5aff69
 DIST kvantum-0.15.2.tar.gz 2652167 BLAKE2B 610bd705d40605cf7d14ffe9acc62bcd276159ffd6f12c315871d633be74d986efc6c45ac938b26851af78565140e2a0dc0424174c543a0622dbf9c951225668 SHA512 98b83a14c3c86636acb6f2440d64dc47496536b5487dc08aed8e8370d656e8838d00866bfb54a4b5752135bdc3d3da0eaab209f5ca3e33c237aada59640a78f7
+EBUILD kvantum-0.14.1.ebuild 841 BLAKE2B dc3c3abc812bfd579bf7171d747c1714d7ec32413ed4a38267287dbd9b0144c193cca3bcb0799fff19afc25a825dbf2467745ddb5e61b87add8dd8f1226a9aba SHA512 091313fec78a21c118a6c8de0244de9aa3e265668045e86b8463aa09ee843fd0e805e50e6fdeafe18cdb212fdfea9bd84e83c77f454e3e129a585d9e431fc5e0
```
Comment (ajakk) [inline x11-themes/kvantum/Manifest:4]: You should configure your repository to use thin manifests. These *.ebuild files caused CI breakage.

https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests

Comment (xbjfk) [reply x11-themes/kvantum/Manifest]: Fixed, thanks

File: x11-themes/kvantum/metadata.xml (hunk 1 above)
Comment (xbjfk) [reply x11-themes/kvantum/metadata.xml]: Fixed, thanks

**Comment by whyman** [comment]
I personally dont see the need for this? All to save what a couple of megabytes?

**Comment by xbjfk** [comment]
True, I expected that they would take up more space. I'll close this PR.

**Pull request closed without merging**